.github/workflows/release.yml

```diff
@@ -105,18 +88,18 @@ jobs:
         shell: bash
         run: |
           release_version="${{ inputs.release_version }}"
+          sed -i "s/\${RELEASE_VERSION}/${{ inputs.release_version }}/g" ${{ env.MANIFEST_PATH }}
           
-          sed -i "s/\${RELEASE_VERSION}/$release_version/g" ${{ env.MANIFEST_PATH }}
-          
-          major_version=$(echo "$release_version" | cut -d '.' -f1,2)
+          major_version=$(echo "${{ inputs.release_version }}" | cut -d '.' -f1,2)
           sed -i 's/\${SHORT_VERSION}/'"$major_version"'/g' ${{ env.MANIFEST_PATH }}
           
-          no_build_nbr_version=$(echo "$release_version" | cut -d '-' -f1)
+          no_build_nbr_version=$(echo "${{ inputs.release_version }}" | cut -d '-' -f1)
           sed -i 's/\${MAJOR_VERSION}/'"$no_build_nbr_version"'/g' ${{ env.MANIFEST_PATH }}
         
           cat ${{ env.MANIFEST_PATH }}
 
       # expecting the release_version to be something like 9.5.1.0-23
+     
```

Comment:
Keeps the is piece of code as it was. See the unnecessary changes
![image](https://github.com/pentaho/actions-common/assets/13484152/6603a7ac-ee70-4f4e-931d-23e880a8b9a3)

Reply:
kept the code same